tools/docker-compose-minikube/minikube/templates/rbac.yml.j2

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: awx-devel
+  namespace: default
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ minikube_service_account_name }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Not a fan of this. Let's be very explicit and mindful about which permissions we need, even in the dev env. This prevents surprises later. https://github.com/ansible/awx-operator/blob/devel/roles/installer/templates/service_account.yaml.j2

Reply:
yeah we used the default SA and gave it cluster-admin because we just wanted to try something fast, but creating a dedicated SA would be ideal. if it is not in the default namespace we have to modify the ContainerGroup podspec.

Reply:
Yes, that makes sense. Thanks @shanemcd